const.go

```diff
@@ -149,18 +151,20 @@ var ConfToEnv = map[string]string{
 	// SubscribeEnabled indicates whether enable the subscription to the Message Queue
 	"messagequeue.subscribe-enabled": "core-data,device-virtual/MESSAGEQUEUE_SUBSCRIBEENABLED",
 
+	// FIXME: does this still exist in Ireland???
```

Comment:
Yes - https://github.com/edgexfoundry/go-mod-bootstrap/blob/3f1fd88ab260e30c6960670415b9eaed3e09b56a/config/types.go#L176

```
// Provides additional configuration properties which do not fit within the existing field.	// Typically the key is the name of the configuration property and the value is a string representation of the	// desired value for the configuration property.	Optional map[string]string
```

So this comment should probably still be here - in case we will support it at some point